tests/ExtraTests/CMakeLists.txt

```diff
@@ -137,6 +137,18 @@ set_tests_properties(CATCH_CONFIG_BAZEL_REPORTER-1
     LABELS "uses-python"
 )
 
+# We must now test this works without the build flag.
+add_executable( BazelReporterNoCatchConfig ${TESTS_DIR}/X30-BazelReporter.cpp )
+target_link_libraries(BazelReporterNoCatchConfig Catch2_buildall_interface)
```

Comment:
This doesn't need the buildall interface, as it explicitly does not touch the compile-time config.

Just linking against `Catch2WithMain` should be enough.